spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -106,6 +106,8 @@
                 getSystemDefault("ZEPPELIN_SPARK_USEHIVECONTEXT",
                     "zeppelin.spark.useHiveContext", "true"),
                 "Use HiveContext instead of SQLContext if it is true.")
+            .add("zeppelin.default.interpreter", "spark",
```

Comment:
The user can add this setting to any interpreter group they want. We have just provided a default for the spark group.
For any other group, the user can still set the option 'zeppelin.default.interpreter' to any interpreter of that group.